packages/syft/src/syft/service/user/user_service.py

```diff
@@ -180,9 +182,9 @@ def search(
             ]
 
             # Return the proper slice using page_index
-            results = results[page_index]
-
-            results = UserViewPage(users=results, total=total)
+            if page_index:
```

Comment:
`page_index` could be 0 which would be falsy here. Better to check the logic and see whether `if page_index is None` is more appropriate.

Reply:
Fixed. Thanks @kiendang